### DISCUSSION TITLE
英語版と日本語版のnotesの比較について

### DISCUSSION THREAD

**Discussion by qq542vev** [ponjbogri/jvs_ja#3] [General]
guskantさんによりご提供いただいた[2016年のgimste, ma'osteのデータ](https://guskant.github.io/lojbo/gismu-cmavo.html)を使用して、notesに含まれる他の単語へのリンクについて比較致しました。

結果は長くなる故、Pastebinにアップロード致しました。

<https://pastebin.com/b5vjBA4k>

該当した単語の量が多いため、全ての単語についてjbovlasteで実際に目視で確認できてはいないのですが、一部を確認した限りでは、英語版にのみに存在する単語リンクは、日本語版ではnotesによる単語の説明が省かれている、または関連語のリストが異なることが多いように感じました。

一方で日本語版のみ存在する単語リンクは、英語版では関連語が掲載されていないが、日本語版では関連語が掲載されている(例えばcenba)、se, teなどもリンクとしているなどの有用なものが多く確認できました。

英語版に100%協調するのが全てではありませんし、英語版のnotesに誤りが確認出来た故に、説明や関連語に含めなかった等のこともあるかとは思われるのですが、英語版と比較して日本語版に不足している部分があるならば、追加していただきたいというのが、個人的な希望です。

また、調査するなかで誤りではないかと思われる単語リンクも確認できました。これらは有用なリンクとして機能していないようです。

~~~~
pinsi {pinsi is the more general term over penbi}
semto {xrabo/xebro}
sraku {sraku does not imply material is removed}
ki'a { mi penmi ko'a } { ko'a ki'a }
~~~~

また関連語のリストの中に、その当該単語が含まれているものもありました。

~~~~
bindo 英語版でも同様
cenba
drudi
merli
minli 英語版でも同様
mlana 英語版でも同様
~~~~

これらについてはできることならば、修整するのが望ましいと思われます。

どなた様か、お手すきの際にでも修整していただけないでしょうか。何卒宜しくお願い致します。

**Comment by niftg** [comment]
jarcoのnoteの「物を見せることについてはarcoを使って」の件は別に真に受けなくていい気がするんですよね…誰が書換したのかわからないですし、当初のtu'aを使用する方法が消されてるしで

**Reply by qq542vev** [reply]
ご返信頂きまして誠にありがとうございます。

当方もこの件については全く同じ気持ちです。仰る通りものの場合はtu'aを使うのが一番かと思われます。

単語の説明について英語版と何処まで協調するかというのは、何れ問題になると感じておりました。

英語版の定義には「officialdata」の名が冠されていますが、実際のところ登録ユーザーであれば誰でも編集出来るようですし、英語版が編集の合わせて、日本語を追従していくのは、容易ではないと思っています。

結局のところ単語毎に英語版からの説明を取捨選択するのがベストでしょうか。

**Comment by qq542vev** [comment]
事後報告で申し訳ないのですが、pinsiを除く9つvalsiについて文面を修整しました。
